assets/multiqc_config.yaml

```diff
@@ -1,34 +1,34 @@
 custom_logo: ../../../docs/images/nf-core_sarek_logo.png
 custom_logo_url: https://github.com/nf-core/sarek/
```

Comment:
I think that the content of this file can be dumped into the newly created `multiqc_config.yml` that should replace this one